cmd/manager/main.go

```diff
@@ -12,7 +12,7 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 	"k8s.io/client-go/rest"
 
-	dv_config "github.com/app-sre/deployment-validation-operator/config"
+	dvConfig "github.com/app-sre/deployment-validation-operator/config"
```

Comment:
```suggestion
	dvconfig "github.com/app-sre/deployment-validation-operator/config"
```

The Go preference for package names is all lowercase rather than camel case or snake case.